src/pybind/cephfs/setup.py

```diff
@@ -117,7 +117,8 @@ def check_sanity():
             extra_preargs=['-iquote{path}'.format(path=os.path.join(CEPH_SRC_DIR, 'include'))]
         )
 
-        if ldflags := os.environ.get('LDFLAGS'):
+        ldflags = os.environ.get('LDFLAGS')
```

Comment:
thank you for the fix. i would suggest to put more details in the commit message instead just claiming it a "syntax error". please note, this is a new syntax introduced in python 3.8, see https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions . so, would be better to amend the commit to explain that we have to support RHEL8, whose default python is python 3.6.8.

Reply:
@tchaikov, It seems  the shaman CentOS8 builders, which serve as the base for container images, aren't current with the ":=" syntax.

Reply:
yeah, and my suggestion still holds. could also add this information to the commit message though.

Reply:
Sounds good, amended the commit message to clarify that the purpose of the change is to support RHEL8, whose default python is python 3.6.8.